src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -52,12 +53,12 @@ class WorkspaceReimburseView extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            unitID: lodashGet(props, 'policy.customUnit.id', ''),
-            unitName: lodashGet(props, 'policy.customUnit.name', ''),
-            unitValue: lodashGet(props, 'policy.customUnit.value', 'mi'),
-            rateID: lodashGet(props, 'policy.customUnit.rate.id', ''),
-            rateName: lodashGet(props, 'policy.customUnit.rate.name', ''),
-            rateValue: this.getRateDisplayValue(lodashGet(props, 'policy.customUnit.rate.value', 0) / 100),
+            unitID: lodashGet(props, 'policy.customUnits.distance.customUnitID', ''),
```

Comment:
i think you missed updating the customUnit shape [here](https://github.com/Expensify/App/blob/0044394d7c741d6eb10c84ee00e853eba197acea/src/pages/workspace/reimburse/WorkspaceReimburseView.js#L36-L45)